tools/src/main/java/org/apache/kafka/trogdor/workload/ProduceBenchSpec.java

```diff
@@ -72,6 +75,8 @@
     private final TopicsSpec activeTopics;
     private final TopicsSpec inactiveTopics;
     private final boolean useConfiguredPartitioner;
+    private final boolean ignoreProduceErrors;
+    private final int topicVerificationRetries;
```

Comment:
It would be better to call this "topicVerificationTries" since 0 is then an unreasonable value.  For REtries, 0 is a reasonable value and not something that we should assume is equivalent to "use the default."